### PR TITLE
Add  products checkout

### PR DESCRIPTION
**Pull request by akv-mshin**
Added amazon checkout for demo purposes

### CODE REVIEW

File: frontend/src/components/ProductCreator/ProductCreator.js [hunk 1]
```diff
@@ -49,13 +50,20 @@ const ProductCreator = ({ categoryId }) => {
...
   const [isError, setIsError] = useState(false);
+  const [scrapeProduct] = useScrapeProduct();
 
   function onProductTextChange(event) {
     setProductLinkInputText(event.target.value);
     if (event.target.value === '') setIsError(false);
     setIsProductLoading(true);
     const linkUrl = event.target.value;
     if (isUrl(linkUrl)) {
+      if (linkUrl.includes('amazon')) {
```
Comment (foreverest) [inline frontend/src/components/ProductCreator/ProductCreator.js]: 🔕 might want better determination of amazon links

**Review by foreverest — APPROVED** [review]
🎉

File: frontend/src/components/ProductCreator/ProductCreator.js (hunk 1 above)
Comment (akv-mshin) [reply frontend/src/components/ProductCreator/ProductCreator.js]: I can put this after we scrape the product to postgre. This way we'll know for sure if this is an amazon product.

Comment (akv-mshin) [reply frontend/src/components/ProductCreator/ProductCreator.js]: done

**Pull request merged**